spring-kafka/src/main/java/org/springframework/kafka/listener/FailedRecordTracker.java

```diff
@@ -150,7 +150,7 @@ public void setRetryListeners(RetryListener... listeners) {
 		this.retryListeners.addAll(Arrays.asList(listeners));
 	}
 
-	List<RetryListener> getRetryListeners() {
+	protected List<RetryListener> getRetryListeners() {
```

Comment:
The class itself is package-protected. Why did you make this method `protected.`?

Reply:
thanks for you review, package-protected is much better.